pkg/rtcp/receiver_report.go

```diff
@@ -67,7 +72,10 @@ func (r ReceiverReport) Marshal() ([]byte, error) {
 		return nil, errTooManyReports
 	}
 
+	rawPacket = append(rawPacket, r.ProfileExtensions...)
```

Comment:
I can't find the right section in the spec... do these extensions contribute toward the Length in the packet header? If so you'll have to update `len()`

Reply:
You're right, i'll dig through the spec and figure out how to do that right (and what to do about padding, etc...)